v63004/u_inref/gtm8791.exp

```diff
@@ -22,5 +20,11 @@ expect -exact "LKE>"
 send -- "\x1A\r"
 
 expect -exact "Suspended (signal)"
+send -- "fg\r"
+
+expect -exact "LKE>"
+send -- "exit\r"
```

Comment:
There are two exit\r usages here. Would help if there is a comment indicating one is to exit from LKE and the other is to exit from the shell. Thanks.